### DISCUSSION TITLE
Ability to see data further back

### DISCUSSION THREAD

**Discussion by abussing** [yhype/yhype-feedback#30] [Ideas]
Hello first of all I love the service!

I was wondering if it's possible to see the traffic data going back further than the ~ 70 days shown on the page?

**Comment by antonkomarev** [comment]
@abussing Good question. Right now it's limited by interface, but we are keeping all data. Will be available in the future.

**Comment by abussing** [comment]
Hi thank you so much for your response.

I was wondering how can I set up yhype for an organization? My advisor is the owner of an organization but when she tried to sign up the same way I did, it didn't show the organization it just showed her personal repositories etc.?

Thanks,
Anderson

________________________________

**Reply by antonkomarev** [reply]
From the [FAQ](https://yhype.me/faq):

**Why my organization is not listed in sidebar?**
You have to be an Owner of the organization. Only owners have access to the statistics.

We cannot collect organization data if you are private member. Go to the "People" tab in organization page on GitHub and change the visibility of membership to Public.

**Comment by abussing** [comment]
Hello,

Thanks so much for your response. My advisor is the owner but when we tried to sign her up it didn't show up. Is it because the people tab in the organization is not visible to the public like you said?

thanks again so much,

best regards,
Anderson
________________________________
